divviup/src/test/java/org/divviup/android/JanusIntegrationTest.java

```diff
@@ -51,15 +51,16 @@
 public class JanusIntegrationTest {
     /** @noinspection SpellCheckingInspection */
     private static final DockerImageName JANUS_INTEROP_AGGREGATOR = DockerImageName.parse(
-            "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_aggregator:0.7.0-prerelease-4@sha256:e85342f168e2cb202bd1eeb7941bd81facb333fb4f114547508e6f9ff82e3da2"
+            "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_aggregator:0.7.2@sha256:5abe7a37896d51121235a47181a271b64aa3975d0d849278567606217cda38a1"
     );
     /** @noinspection SpellCheckingInspection */
     private static final DockerImageName JANUS_INTEROP_COLLECTOR = DockerImageName.parse(
-            "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_collector:0.7.0-prerelease-4@sha256:617df823da7efe5566eb53e4cbdbe983b027b2369ad430b4fa7de6e972353488"
+            "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_collector:0.7.2@sha256:3004724d6f2f3c40763022a27202ff88e0a08fcde0176d6b8343bb9948231ce5"
     );
     private static final int BASE64_FLAGS = Base64.NO_PADDING | Base64.NO_WRAP | Base64.URL_SAFE;
     private static final int TIME_PRECISION_SECONDS = 3600;
     private static final String LEADER_ALIAS = "leader", HELPER_ALIAS = "helper";
+    private static final String RUST_LOG = "debug";
```

Comment:
I suggest we set the default log filter to `info`. `debug` will pick up a lot more (including Postgres statements), so it would result in a lot more spam in the terminal, and more time tar-ing, untar-ing, and printing them when tests are passing.

Reply:
Fair enough.

I've also tied RUST_LOG to the system's environment variable, so we can `RUST_LOG=debug ./gradlew check` to get the verbose logging without having to modify the source.